.github/workflows/windows-installer.yml

```
@@ -0,0 +1,54 @@
+name: "Windows Installer"
+
+# any branch is useful for testing before a PR is submitted
+on: [push, pull_request]
+
+jobs:
+  build:
+    # run on pushes to any branch
+    # run on PRs from external forks
+    if: |
+         (github.event_name != 'pull_request')
+         || (github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id)
+    name: Building Windows Zip and Executable Installer on Linux without plugins
```

Please keep this brief:
```suggestion
    name: Build
```